packages/plugin-eslint/src/lib/eslint-plugin.ts

```diff
@@ -36,19 +37,23 @@ export async function eslintPlugin(
   config: ESLintPluginConfig,
   options?: ESLintPluginOptions,
 ): Promise<PluginConfig> {
-  const targets = eslintPluginConfigSchema.parse(config);
+  const configPath = fileURLToPath(path.dirname(import.meta.url));
+
+  const targets = parseSchema(eslintPluginConfigSchema, config, {
+    schemaType: 'ESLint plugin config',
+    sourcePath: configPath,
+  });
 
   const customGroups = options
-    ? eslintPluginOptionsSchema.parse(options).groups
+    ? parseSchema(eslintPluginOptionsSchema, options, {
+        schemaType: 'ESLint plugin options',
+        sourcePath: configPath,
+      }).groups
     : undefined;
```

Comment:
I believe the `configPath` would be something like `node_modules/@code-pushup/eslint-plugin/src/lib/eslint-plugin.js`, which isn't where the invalid config is defined or where the user should fix it. If we could point them somewhere, it would be to their `code-pushup.config.ts`, but I don't think we can tell where the `eslintPlugin` is being called from.

I think it's simplest to make the `sourcePath` optional, since we won't always be able to associate an invalid schema with which file it was defined in.